build-tools/src/main/java/uk/ac/ox/ndph/mts/json_schema_generator/Main.java

```diff
@@ -3,27 +3,61 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
-import uk.ac.ox.ndph.mts.json_schema_generator.model.TrialConfigFile;
+import com.google.common.collect.Lists;
+import uk.ac.ox.ndph.mts.json_schema_generator.model.InitTrialConfig;
+import uk.ac.ox.ndph.mts.json_schema_generator.model.PractitionerConfig;
+import uk.ac.ox.ndph.mts.json_schema_generator.model.SiteConfig;
+import uk.ac.ox.ndph.mts.json_schema_generator.model.TrialDefinitionFile;
 
 import java.io.File;
 import java.io.IOException;
+import java.util.ArrayList;
 
 
 public final class Main {
-    static String fileName = "trialConfigJsonSchema.json";
+
+    private static class GeneratedTuple {
+        private Class cls;
+        private String filename;
+
+        public Class getCls() {
+            return cls;
+        }
+
+        public String getFilename() {
+            return filename;
+        }
+
+        public GeneratedTuple(Class cls, String filename) {
+            this.cls = cls;
+            this.filename = filename;
+        }
+    }
 
     public static void main(final String[] args) throws IOException {
+        ArrayList<GeneratedTuple> itemToProcess = Lists.newArrayList(
+            new GeneratedTuple(TrialDefinitionFile.class, "definition-schema.json"),
+            new GeneratedTuple(InitTrialConfig.class, "init-service-trial-schema.json"),
+            new GeneratedTuple(PractitionerConfig.class, "practitioner-service-configuration-schema.json"),
+            new GeneratedTuple(SiteConfig.class, "site-service-configuration-schema.json"));
```

Comment:
Since the classes are anyway on this module, what do you think about a property for the schema name and "ignore" it in the serialization?